ppr-ui/src/utils/date-helper.ts

```diff
@@ -18,11 +18,12 @@ export function format12HourTime (date: Date): string {
 
 /** Converts date to display format. */
 export function convertDate (date: Date, includeTime: boolean, includeTz: boolean): string {
-  if (!includeTime) return moment(date).format('MMMM D, Y')
+  // For just dates use the UTC timestamp otherwise the day of month is incorrect (1 day earlier).
+  if (!includeTime) return moment(date).utc().format('MMMM D, Y')
 
   // add 'Pacific Time' to end if pacific timezone
   let timezone = ''
-  if ((date.toString()).includes('Pacific')) timezone = 'Pacific time'
+  if ((date.toString()).includes('Pacific')) timezone = 'Pacific Time'
```

Comment:
Change this back pls. Lower case 't' is correct (I did the same thing before too and was corrected)

Reply:
Sorry, restored to Pacific time